samples/browser/src/main/java/org/mozilla/samples/browser/SampleApplication.kt

```diff
@@ -26,6 +27,13 @@ class SampleApplication : Application() {
             return
         }
 
+        // IMPORTANT: the following lines initialize the Glean SDK but disable upload
+        // of pings. If, for testing purposes, upload is required to be on, change the
+        // next line to `Glean.setUploadEnabled(true)`
+        Glean.setUploadEnabled(false)
+
+        Glean.initialize(applicationContext)
```

Comment:
Since you mention this is for Gecko engineers: We do not setup the GleanAdapter?

Reply:
Good point, thanks for flagging this, I forgot about that. I asked for review again, since I touched a bunch of new files.